src/Stryker.TestRunner/Tests/TestGuidsList.cs

```diff
@@ -0,0 +1,89 @@
+using Stryker.Abstractions.Testing;
+using System.Collections.Generic;
+using System;
+using System.Linq;
+
+namespace Stryker.TestRunner.Tests;
+
+public class TestGuidsList : ITestIdentifiers
```

Comment:
Isn't this class duplicate?

Reply:
Not exactly. But I'll see if I can make the class generic to reuse it